src/services/OpenAPIParser.ts

```diff
@@ -173,7 +173,7 @@ export class OpenAPIParser {
       keys.some(k => k !== 'description' && k !== 'title' && k !== 'externalDocs')
     ) {
       return {
-        allOf: [rest, resolved],
+        allOf: [resolved, rest],
```

Comment:
Is it related to our recent changes in all of merging?

Reply:
Yes